lib/virtfs/ntfs/utils.rb

```diff
@@ -39,5 +39,13 @@ def self.validate_signature(signature, expected)
         raise "Invalid Signature <#{signature}>"
       end
     end
+
+    def self.UnicodeToUtf8(string)
+      string.dup.force_encoding('UTF-16LE').encode('UTF-8')
+    end
+
+    def self.AsciiToUtf8(string)
+      dup.force_encoding('ISO-8859-1').encode('UTF-8')
```

Comment:
Think this was a typo.  Also since these methods are standalone, can you add specs for them?

```suggestion
      string.dup.force_encoding('ISO-8859-1').encode('UTF-8')
```